src/browser/api/window.js

```diff
@@ -502,6 +502,25 @@ Window.create = function(id, opts) {
             }
         });
 
+        var emitToAppAndWin = (...types) => {
```

Comment:
I'll but a check before triggering the event routed _to_ the app. Is that what you mean?